common/src/main/resources/tachyon-default.properties

```diff
@@ -71,6 +71,7 @@ tachyon.worker.network.netty.watermark.high=32KB
 tachyon.worker.network.netty.watermark.low=8KB
 tachyon.worker.allocate.strategy.class=tachyon.worker.block.allocator.MaxFreeAllocator
 tachyon.worker.evict.strategy.class=tachyon.worker.block.evictor.LRUEvictor
+tachyon.worker.balance.strategy=BALANCE
```

Comment:
is this a worker policy? or a policy of the client/user?   so maybe should be `tachyon.user` 
nit: instead of calling it `balance.strategy`, maybe  `write.strategy`?